### ISSUE TITLE
jpm4j.org down

### ISSUE THREAD

**Issue by bherr2** [jpm4j/general#2]
The jpm4j website looks like the hosting has expired. Is this project dead?

[Screenshot](https://screenshots.firefox.com/eLqvS4XmOQi5DB9o/ww12.jpm4j.org)

**Comment by pkriens** [comment]
Yes, it was retired about 2 years ago due to lack of enthusiasm from the public. This was discussed in the enroute and bndtools mailing lists after replacements were created.

**Comment by bherr2** [comment]
Ahh, ok. I figured as much. I got here by trying to install the CLI version of bnd, which still references jpm (<https://bnd.bndtools.org/chapters/120-install.html>). A note on the repos that the project is defunct would be appreciated. And another way to easily install the bnd CLI would be great too (I eventually got it by building the bndtools repo).

(Unrelated: I'm a big fan of your work :+1: )

**Comment by pkriens** [comment]
I added a message on the repo, felt kind of sad. 

Liferay took over the jpm command line since it also installs Jars with a Main-Class. See https://github.com/bndtools/jpmcli
